src/config/util.js

```diff
@@ -55,16 +58,18 @@ function getPlugins(plugins) {
   if (typeof plugins !== 'object' || !Array.isArray(plugins)) return undefined;
 
   return plugins
+    .filter(Boolean)
```

Comment:
What's this bit for?

Reply:
It drops falsey values `_.uniq` but built-in. We are concating possibly falsey values in other steps of the pipeline. Best to just do the coercing here I think?

Reply:
* `_.compact` I mean

Reply:
Cool, thanks!